test/webgl/context-state.spec.js

```diff
@@ -65,3 +66,20 @@ test('WebGLState#withState', t => {
   t.end();
 });
 
+test('WebGLState#resetContext', t => {
+  const {gl} = fixture;
+
+  setParameter(gl, 'clearColor', [0, 1, 0, 1]);
```

Comment:
For another PR: Would be great to add a complete list of alternate values here so we could test all settings.